src/coreclr/src/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -108,7 +108,7 @@ set(CORECLR_LIBRARIES
     gcinfo # Condition="'$(TargetCpu)'=='amd64' or '$(TargetCpu)' == 'arm' or '$(TargetCpu)' == 'arm64'"
     ildbsymlib
     utilcode
-    v3binder
```

Comment:
This was duplicated on the list.

Reply:
The duplication was intentional. See the comment above. The Linux linker works differently from the one on Windows.  It goes over the list of libraries from left to right, remembering undefined symbols. When it processes a new library from the list, it checks whether it resolves some of the unresolved symbols and if it does, includes the related objects from the library. So if there are circular references, libraries need to be duplicated or the START_LIBRARY_GROUP / END_LIBRARY_GROUP macro needs to be used around set of libraries with circular reference. Not doing this properly would lead into libcoreclr.so that would compile ok, but it would have unresolved symbols and fail to load.
However, it is possible that due to various cleanups from the past, this duplication is not needed anymore.

Reply:
I see. I'll put it back then.